CollAction/Controllers/ProjectsController.cs

```diff
@@ -136,7 +136,10 @@ public ViewResult StartInfo()
                 Goal = model.Goal,
                 Proposal = model.Proposal,
                 CreatorComments = model.CreatorComments,
-                CategoryId = model.CategoryId,
+                CategoryId = _context
+                    .Categories
+                    .FirstOrDefault(c => c.Name == "Friesland")
```

Comment:
* SingleAsync() is probably the method that makes the most sense here:
  + Single because you want it to return exactly one record (https://stackoverflow.com/questions/3485317/entity-framework-4-single-vs-first-vs-firstordefault)
  + Async because that's more efficient for IO. Doesn't matter much here, but it's a good habit to use async to access the database